bqskit/runtime/base.py

```diff
@@ -82,7 +104,13 @@ def send_outgoing(node: ServerBase) -> None:
 
         outgoing[0].send((outgoing[1], outgoing[2]))
         node.logger.debug(f'Sent message {outgoing[1].name}.')
```

Comment:
It would be benefitial to also indicate to where the message is sent up\down, and if down to which id (if possible)